modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -56,6 +58,14 @@ def __init__(self, object_id, length=None, width=None, ip=None, call_queue=None)
         self._length_cache = length
         self._width_cache = width
         self._ip_cache = ip
+        self._identity = uuid.uuid4().hex[8:]
```

Comment:
Why are the changes made only for Ray? What about Dask?

Reply:
@naren-ponder I think these changes could be left off this PR as well, as with the Ray remote logs.